rag/svr/task_executor_refactor/dataset_structure_merger.py

```
@@ -14,143 +14,662 @@
+                    {"id": [rid]},
+                    {"doc_ids_kwd": remaining},
+                    index,
+                    kb_id,
+                )
+            except Exception:
+                logging.exception(
+                    "structure_merge: failed to update doc_ids_kwd on row %s",
+                    rid,
+                )
+
+    # ── Pass 3: clear processed meta rows ────────────────────────────
```

Ruff (0.16.0)

[warning] Comment contains ambiguous ` ` (NARROW NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF003)